platform/solr/solr-dependencies/pom.xml

```diff
@@ -23,6 +23,13 @@
     <name>DDF :: Platform :: Solr :: Dependencies</name>
     <description>Shaded jar of Solr core dependencies.</description>
     <packaging>jar</packaging>
+    <properties>
+        <io.dropwizard.metrics.version>3.1.2</io.dropwizard.metrics.version>
+        <info.ganglia.gmetric4j.version>1.0.7</info.ganglia.gmetric4j.version>
+        <org.slf4j.version>1.7.7</org.slf4j.version>
```

Comment:
We can work that under a different ticket since this is focused on the Solr upgrade